### DISCUSSION TITLE
Named Weeks

### DISCUSSION THREAD

**Discussion by GolaWaya** [vigoren/foundryvtt-simple-calendar#349] [Ideas]
Absolutely love what you guys have done, this has become a must for me in my games.  I have a rather odd time system in one game I use,  Intead of naming the days, they name the weeks (each month has 6 weeks except for one which has 7) and they would call it 4th Weekname of Month (for example 4th Trail of Chesun) which would be the 4th day on the 5th week of the month.  Would be awesome to have week naming support like you do with years.

**Comment by claycle** [comment]
Named weeks would be great, please. Everything *else* is customizable, after all.
